sdk/data/aztables/autorest.md

```diff
@@ -11,7 +11,7 @@ clear-output-folder: false
 output-folder: internal
 tag: package-2019-02
 credential-scope: none
-use: "@autorest/go@4.0.0-preview.36"
+use: "C:/github.com/autorest.go"
```

Comment:
This is an odd path, is this a correct thing to do? (ie, shouldn't there be some version pining happening here?)

Reply:
Once the latest autorest version is pinned I will update this